ecmd-core/ext/template/capi/makefile.capi

```diff
@@ -10,6 +10,7 @@ EXTENSION_NAME_u1 := $(shell perl -e 'printf(ucfirst(${EXTENSION_NAME}))')
 CAPI_INCLUDES := ${CAPI_INCLUDES} ${EXTENSION_NAME}ClientCapi.H ${EXTENSION_NAME}Structs.H
 INT_INCLUDES  := ${INT_INCLUDES} ecmdClientCapi.H ecmdDataBufferBase.H ecmdDataBuffer.H 
 INT_INCLUDES  := ${INT_INCLUDES} ecmdReturnCodes.H ecmdStructs.H ecmdUtils.H ecmdClientEnums.H
+EXTRA_INCLUDES := ${SRCPATH}/ecmdDllCapi.H ${SRCPATH}/cipDllCapi.H
```

Comment:
could we instead do 
```
CAPI_INCLUDES := ${CAPI_INCLUDES} ${EXTENSION_NAME}DllCapi.H
```

Reply:
and not have to add EXTRA_INCLUDES?

Reply:
And the `ecmdDllCapi.H` copy needs to not be in the template make file.  It should be in the makefile that actually generates it (the main capi one I believe).

Reply:
{EXTENSION_NAME} points to cmd or cip but the header file named ecmdDllCapi.H or cipDllCapi.H (Only this one matches)

Reply:
Please check if the current changes look good or not. it did the work

Reply:
There is also and optional ecmd-internal sub-repo which has more extensions that can be configured and compiled/installed.  Using ${EXTENSION_NAME} would account for all of these as well.

Reply:
This is the configure option i am chosing "./config.py --without-swig  --target arm --extensions "cmd cip" --build-verbose" so i am getting cmd or cip and not ecmd

Reply:
I have updated it as per the comments